client/builder.sh

```diff
@@ -2,6 +2,7 @@
 echo "BUILDING FRONTEND FOR PRODUCTION"
 npm install
 npm install -g polymer-cli --unsafe-perm
+npm install -g workbox-cli
```

Comment:
do we need to install it twice?

Reply:
the other one didn't work. I'll delete it.